src/main/scala/org/broadinstitute/dsde/firecloud/utils/TSVFormatter.scala

```diff
@@ -95,13 +96,12 @@ object TSVFormatter {
     * @return Ordered list of rows
     */
   def makeMembershipRows(entityType: String, entities: Seq[Entity]): Seq[IndexedSeq[String]] = {
-    val memberPlural = pluralizeMemberType(memberTypeFromEntityType(entityType))
     entities.filter { _.entityType == entityType }.flatMap {
       entity =>
         entity.attributes.filter {
           // To make the membership file, we need the array of elements that correspond to the set type.
           // All other top-level properties are not necessary and are only used for the data load file.
-          case (attributeName, _) => attributeName.equals(AttributeName.withDefaultNS(memberPlural))
+          case (attributeName, _) => attributeName.equals(AttributeName.withDefaultNS(FlexibleModelSchema.pluralizeEntityType(FlexibleModelSchema.memberTypeFromEntityType(entityType).get)))
```

Comment:
It looks to me like `getOrElse(entityType)` would most closely mimic the previous behavior:
```
  def memberTypeFromEntityType(entityType: String): String = ModelSchema.getCollectionMemberType(entityType).get.getOrElse(entityType.replace("_set", ""))
```
This previously looked up the entity type in the schema, throwing an exception if missing. Now, we want any entity type to be valid because there's no schema. However, we might still get `None` from `FlexibleModelSchema.memberTypeFromEntityType(entityType)` if the name doesn't end in `_set`. Previously, as long as it was a valid type, we'd trim `_set` from the name if it's in there, otherwise leave the name alone. Therefore `getOrElse(entityType)`.